tasks/benchmark-install-against-revisions/src/fixtures/install.bash

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+cd "$(dirname "$0")"
+exec ./pacquet/target/release/pacquet install
```

Comment:
`just cli install` does the same thing

Reply:
In this case, it doesn't. The `install.bash` is to be first installed by the benchmarking program to the benchmarking place, then to be executed by `hyperfine`. Also, I must make sure it run the `release` version, adding `just` as a middleman would just overcomplicate things for no benefit.